mock_bitaxe.py

```
@@ -0,0 +1,87 @@
+    # no-op for the mock; the client sleeps on its side
+    return jsonify({"status": "restarting"})
+
+
+# Optional: legacy endpoint (not used by the benchmark script)
+@app.get("/api/hashrate")
+def hashrate():
+    return jsonify({"five_sec": 0.0, "one_min": 0.0, "five_min": 0.0})
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=5000)
```

Ruff (0.15.11)

[error] Possible binding to all interfaces

(S104)